src/containers/Routes/components/Drawer/Drawer.tsx

```diff
@@ -1,12 +1,13 @@
 import React, { Fragment } from "react";
 import { TouchableOpacity } from "react-native";
-import { WrapperContainer, Typography, Divider } from "../../../../components";
+import { SafeAreaView } from "react-native-safe-area-context";
+import { Typography, Divider } from "../../../../components";
 import { drawerContent } from "./Drawer.helpers";
 import { styles } from "./Drawer.styles";
 
 export const DrawerMenu = () => {
   return (
-    <WrapperContainer>
+    <SafeAreaView style={[styles.mainContainer]}>
```

Comment:
Please use out-of-box solution
https://reactnavigation.org/docs/drawer-navigator#options

Reply:
Will check

Reply:
done